mtest/libmscore/albumsIO/muse2_loaded_ref.msca

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<museScore version="3.01">
+  <Album>
+    <name>Blaues Heft (2.0)</name>
```

Comment:
could you change this to `My Album (2.x)`

Reply:
Done